src/SUFST/src/can_unpack.c

```diff
@@ -42,7 +44,34 @@ unpack_ptr->rx_queue = can_queue;
                                TX_NO_TIME_SLICE,
                                TX_AUTO_START);
     }
+    if (tx_status == TX_SUCCESS)
+    {
+        tx_status = tx_queue_create(&unpack_ptr->rx_queue,
+                    "My Rx Queue",
+                    TX_1_ULONG,
+                    unpack_ptr->rx_queue_mem,
+                    sizeof(unpack_ptr->rx_queue));        
+    }
+    
+    // subscribe to a message
+    if (tx_status == TX_SUCCESS)
+    {
+        for(int i = 0; i < TABLE_SIZE; i++)
+        {
+            tx_status = rtcan_subscribe(&unpack_ptr->rtcan, can_handler_get(i)->identifier , &unpack_ptr->rx_queue);
+            if(tx_status != RTCAN_OK)
+            {
+                break;
```

Comment:
Not sure what we should do if this error occurs? I think we should handle it somehow. In general maybe could create some simple error handling system that turns one of the builtin LEDs red and just goes `while(1)` if there is a fatal error like this.